app/src/test/java/org/simple/clinic/summary/PatientSummaryViewRendererTest.kt

```diff
@@ -279,7 +279,7 @@ class PatientSummaryViewRendererTest {
     // given
     val patientUuid = UUID.fromString("66bc73a4-29a1-4b07-8be1-9223dc5d7cbb")
     val model = PatientSummaryModel.from(
-        openIntention = OpenIntention.ViewExistingPatientWithTeleconsultLog(
+        openIntention = OpenIntention.  ViewExistingPatientWithTeleconsultLog(
```

Comment:
Was this extra space added accidentally?

Reply:
Yes. Will fix it 👍🏽